src/lib/protocols/present-proof/messages/PresentationAckMessage.ts

```diff
@@ -1,19 +1,16 @@
 import { Equals } from 'class-validator';
 
-import { AgentMessage } from '../../../agent/AgentMessage';
+import { AckMessage, AckMessageOptions } from '../../connections';
 import { PresentProofMessageType } from './PresentProofMessageType';
 
-interface PresentationAckMessageOptions {
-  id?: string;
-}
+export type PresentationAckMessageOptions = AckMessageOptions;
```

Comment:
The same question as with `CredentialAckMessageOptions`

Reply:
See answer above